README.md

```diff
@@ -7,8 +7,8 @@ Experimental GraphQL client.
 ### 1) Create a client
 
 Either:
-  * A `GraphQLClient[F[_], S]` (supporting queries and mutations), or
-  * A `GraphQLStreamingClient[F[_], S]` (supporting queries, mutations and subscriptions).
+  * A `TransactionalClient[F[_], S]` (supporting queries and mutations), or
```

Comment:
Is there a benefit of using this over `Streaming`? If e.g. you don't plan to streambut maybe want to keep the option open?

Reply:
Not sure I understood. The renames were:
* `GraphQLClient` (which was a bit broad) -> `TransactionalClient` (meaning client that supports one-shots but not streaming, alternate name suggestions are welcome).
* `GraphQLStreamingClient` -> `StreamingClient`.

In general I tried removing the `GraphQL` prefix as much as possible.

Reply:
I meant of using `TransactionalClient` over `StreamingClient`. does it make sense to have a single one?

Maybe it is about allowing implementations that don't support streaming?

Reply:
Exactly. If you want to do one-shots over HTTP, that would be the one to use.